supabase/functions/upload-to-storage/index.ts

```diff
@@ -131,13 +133,13 @@ serve(async function handler(req: Request) {
       body: [
         [
           "University degree",
-          `${employee.university}\n${employee.degree}\n${employee.university_start} - ${employee.university_end}`,
+          `${employee.education[0].university_name}\n${employee.education[0].degree}\n${employee.education[0].start_year} - ${employee.education[0].end_year}`,
```

Comment:
We will have possibilities for multiple educations per person. For example - Bachelor's degree + Master's degree. Can we display this in the same manner as certifications in L142?